.github/workflows/main.yml

```diff
@@ -3,6 +3,17 @@ name: CI
 on: [push]
 
 jobs:
+  run-frontend-v2-lint:
+    runs-on: ubuntu-latest
+    name: Run linting for frontend v2
+    steps:
+      - uses: actions/checkout@v2
+      - name: install node v12
+        uses: actions/setup-node@v1
+        with:
+          node-version: 12
```

Comment:
do we wanna use v16?

Reply:
the goal is to eventually upgrade everything to v16, i was hoping we could do scrapers to 16 during the rewrite